src/openrct2/config/Config.cpp

```diff
@@ -237,6 +237,7 @@ namespace OpenRCT2::Config
             model->RenderWeatherGloom = reader->GetBoolean("render_weather_gloom", true);
             model->ShowGuestPurchases = reader->GetBoolean("show_guest_purchases", false);
             model->ShowRealNamesOfGuests = reader->GetBoolean("show_real_names_of_guests", true);
+            model->ShowRealNamesOfStaff = reader->GetBoolean("show_real_names_of_staff", true);
```

Comment:
Not sure if this should be `true` by default. Personally, I prefer to have it off, I think.

Reply:
I prefer to have it off too, while peeps names are just often random information, one might be using the staff numbers to memorize where they're assigned to or whatever.

It also should be `false` given that the changelog mentions it is optional